README.md

```diff
@@ -36,6 +36,12 @@ Or you can install by building from source directly as follows. Go toolchain is
 $ go get -u github.com/rhysd/notes-cli/cmd/notes
 ```
 
+If the build fails due to https://github.com/golang/go/issues/30515 then you may have to enable `module mode` as described in `go help env`
+
+```
+go env -w GO111MODULE=on
+```
+
```

Comment:
This changes user's default configuration, right? It may break other projects on the machine. I think specifying `GO111MODULE` on `go get` would be safer and cleaner:

```
$ GO111MODULE=on go get -u github.com/rhysd/notes-cli/cmd/notes
```

Reply:
Yeah, feel free to recommend that. But according to Google's issue this is soon going to be default anyway, so I wouldn't worry about it. Maybe you could also specify both and make the user aware that one permanently modifies the env.

Reply:
Does 96511e5 seem adequate?

Reply:
That also sounds good :+1: